openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMUtils.java

```diff
@@ -82,6 +84,31 @@ static CpuArchitecture getCpuArchitecture(final String cpuArchName) {
         }
     }
 
+    /**
+     * Checks if the current operative system is Alpine.
+     *
+     * @return true when the operative system is Alpine and false otherwise.
+     */
+    static private boolean isAlpine() {
```

Comment:
There are more Linux distributions using MUSL not only Alpine and we should check for that instead.

Reply:
I changed it in the latest commits. Now instead of searching for Alpine in specific, it uses an env variable that must be defined by the user in order to decide which libc implementation to use.